Improve readme (#486)
OpenTermsArchive/p2b-compliance-declarations: README.md

These service declarations are consumed by the [Open Terms Archive engine](https://github.com/OpenTermsArchive/engine).

You can [explore the tracked versions](https://github.com/OpenTermsArchive/p2b-compliance-versions) interactively or [download them as a dataset](https://github.com/OpenTermsArchive/p2b-compliance-versions/releases) in the `OpenTermsArchive/p2b-compliance-versions` repository.

- - -

## Scope

The terms tracked by this repository are those of services that operate:

- In the European Union.
- When available, in English; otherwise in the official language of the country of incorporation of the provider.
- Under Regulation (EU) 2019/1150 of 20 June 2019 on promoting fairness and transparency for business users of online intermediation services (the [P2B Regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=celex%3A32019R1150)), that entered into force on 12 July 2019 and has applied since 12 July 2020.

The aim of this dataset is to enable monitoring of the implementation of the P2B Regulation.

- - -

## Contribute

> This collection does not have a known maintainer, and contributions are thus unlikely to be handled.

Reviewer: ```suggestion
This collection does not have an active maintainer, and contributions are thus unlikely to be handled.
```
